ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -282,6 +305,47 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-usertags-custom-security-groups
+- as: aws-ipi-wavelength-byo-subnet-ovn-day2-f28
+  cron: 5 8 10 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EDGE_ZONE_TYPES: wavelength-zone
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn-day2
+- as: aws-ipi-wavelength-byo-subnet-ovn-mtu-day2-f28
```

Comment:
For upgrade, I think it should be enough that we only need to keep this one, and remove `aws-ipi-wavelength-byo-subnet-ovn-day2-f28` and `aws-ipi-wavelength-byo-subnet-ovn-f28`